### PR TITLE
Implemented: option to turn on/off eCom inventory on the settings page (#141)

### LINKED ISSUE #141: There should be an option to turn off/on the ecom inventory from the settings page
## What is the motivation for adding/enhancing this feature?
There should be an option to turn off/on the ecom inventory for the facility from the settings page like the old Fulfillment app.

## What are the acceptance criteria?

- [ ] ...

## Can you complete this feature request by yourself?

- [ ] YES
- [ ] NO

### PR DESCRIPTION
**Pull request by k2maan**
### Related Issues

Closes #141 

### Screenshots of Visual Changes before/after (If There Are Any)

**IMPORTANT NOTICE** - Remember to add changelog entry

### Contribution and Currently Important Rules Acceptance

- [ ] I read and followed [contribution rules](https://github.com/hotwax/ionic-bopis#contribution-guideline)

### CODE REVIEW

File: src/views/Settings.vue [hunk 1]
```diff
@@ -351,6 +437,40 @@ export default defineComponent({
 
       await alert.present();
     },
+    async turnOnEcomInv() {
+      const alert = await alertController.create({
+        header: this.$t('Turn on eCom inventory for ', { facilityName: this.currentFacility.name }),
+        message: translate('Are you sure you want perform this action?'),
```
Comment (ymaheshwari1) [inline src/views/Settings.vue:443]:
```suggestion
        message: translate('Are you sure you want to perform this action?'),
```

File: src/views/Settings.vue [hunk 2]
```diff
@@ -249,6 +259,45 @@ export default defineComponent({
...
+          "inputFields": {
+            "facilityGroupTypeId": 'SHOPIFY_GROUP_FAC'
+          },
+          "fieldList": ["facilityGroupId", "facilityGroupTypeId"],
+          "viewSize": 1,
+        })
+
+        if (!hasError(resp)) {
+          const facilityGroupData = resp.data.docs[0]
+          resp = await UserService.getFacilityGroupDetails({
```
Comment (ymaheshwari1) [inline src/views/Settings.vue:278]:
```suggestion
          resp = await UserService.getFacilityGroupAndMemberDetails({
```

**Review by ymaheshwari1 — CHANGES_REQUESTED** [review]

File: src/views/Settings.vue [hunk 3]
```diff
@@ -263,7 +263,11 @@ export default defineComponent({
       appVersion: "",
       locales: process.env.VUE_APP_LOCALES ? JSON.parse(process.env.VUE_APP_LOCALES) : {"en": "English"},
       currentFacilityDetails: {} as any,
-      isStoreFulfilmentTurnedOn: true
+      facilityGroupDetails: {} as any,
+      outstandingOrdersCount: 0,
```
Comment (ymaheshwari1) [inline src/views/Settings.vue:267]: Why we have defined the variables for order count?

**Review by ymaheshwari1 — CHANGES_REQUESTED** [review]

File: src/views/Settings.vue (hunk 3 above)
Comment (k2maan) [reply src/views/Settings.vue]: Removed sir

**Pull request merged**